src/notebooklm/_artifacts.py

```
@@ -2472,3 +2473,198 @@ def _is_media_ready(self, art: builtins.list[Any], artifact_type: int) -> bool:
                 e,
             )
             return not is_media  # False for media (continue polling), True for non-media
+
+    # =============================================================================
+    # Bulk Operations
+    # =============================================================================
+
+    async def generate_bulk(
+        self,
+        notebook_ids: list[str],
```

Ruff (0.15.9)

[error] Python builtin is shadowed by method `list` from line 249

(A003)